server/v1/controllers/shows/utils/utils.js

```diff
@@ -159,16 +162,11 @@ function reduceShowsByRepeatProperty(shows, recurringCheckValue) {
 function returnDatesArrayByRepeatRule(show, startDate, endDate) {
   const rule = new RRule(createRRule(show));
   try {
-    let shows = rule.between(new Date(startDate), new Date(endDate), true);
-
-    //the .between() function will not get a currently playing event, so we'll have to check for it
```

Comment:
Removes the use of `before`, as this has the same performance issues that `between` has in the `rrule` library. (The fork I made of rrule only fixes the performance issues for `between`)